src/pages/instances/InstanceOverviewDeviceList.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import { LxdDeviceValue, LxdGPUDevice } from "types/device";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  device: LxdDeviceValue;
+  instance: LxdInstance;
+}
+
+const InstanceOverviewDeviceList: FC<Props> = ({ device, instance }) => {
+  return (
```

Comment:
In general I think using ternary is not very readable when it exceeds 2 conditions since it introduces cognitive load for someone reading the code e.g. trying to find the colons. Instead, we can try use the following structure:

```
if (device.type === "disk") {
    if (isRootDisk(device as FormDevice)) {
          return A;
    }
    
     return B;
}

if (device.type === "gpu) {
    return C;
}

etc...
```

Reply:
None of the other device types, such as gpu, have more than 2 conditions. Should I still change the ternary operator to an if statement?

Reply:
As written below, I understand this is mostly due to an easier return for the Other device types.